rapid7_insightvm_cloud/plugin.spec.yaml

```diff
@@ -0,0 +1,2237 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: rapid7_insightvm_cloud
+title: Rapid7 InsightVM for the Cloud
+description: InsightVM is a powerful vulnerability management tool which finds, prioritizes, and remediates vulnerabilities. This plugin uses the v4 cloud API to view assets, scan results and start scans
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/rapid7_insightvm
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.rapid7.com
+tags:
+- insightvm
+- rapid7
+- vulnerability
+hub_tags:
+  use_cases: [asset_inventory, vulnerability_management]
+  keywords: [insightvm, rapid7, cloud]
+  features: []
+
+types:
+  report_id:
```

Comment:
Doesn't look like all these types are used are they? Perhaps copied form the rapid7_insigthvm plugin?